packages/components/src/components/Text/Text.tsx

```diff
@@ -0,0 +1,118 @@
+import {
+  Text as RNText,
+  TextProps as RNTextProps,
+  TextStyle,
+} from 'react-native'
+import { font } from '@department-of-veterans-affairs/mobile-tokens'
+import React, { FC } from 'react'
+
+import { SpacerSize, getSpacingToken } from '../Spacer/Spacer'
+import { useTheme } from '../../utils'
+
+type TextSizes = 'xs' | 'sm' | 'md' | 'lg'
+type BaseTones = 'default' | 'subtle' | 'inverse'
+type BodyTones = BaseTones | 'error'
+
+type BodyProps = {
+  /** Size: xs, sm, md, or lg. Defaults to 'md' for body and heading */
+  size?: TextSizes
+  /** Text color: default, subtle, inverse, error. Defaults to vadsColorForegroundDefault. */
+  tone?: BodyTones
+  /** Variant: body, heading, display, or navigation */
```

Comment:
Clean up references to `navigation` since it has been removed.

Reply:
Removed